code/obj/submachine/seed.dm

```diff
@@ -1084,26 +1084,110 @@
 	attack_ai(mob/user as mob)
 		return src.Attackhand(user)
 
-	attack_hand(var/mob/user as mob)
-		src.add_dialog(user)
-		var/dat = "<B>[src.name]</B><BR><HR>"
-		dat += "<b>Amount to Vend</b>: <A href='?src=\ref[src];amount=1'>[src.vendamt]</A><br>"
-		if (src.category)
-			dat += "<b>Filter</b>: [src.category] <A href='?src=\ref[src];category=1'>(Clear)</A><br>"
-		else
-			dat += "<b>Filter</b>: <A href='?src=\ref[src];category=1'>(Set)</A><br>"
-		if (!src.can_vend)
-			dat+= "<u>Unit currently out of charge. Please wait.</u><br>"
-		dat += "<br>"
+	proc/getPlantImg(var/datum/plant/plant)
```

Comment:
Similar to note about `plant_base64_cache`, probably want to move out of here if something else ends up using it.

Given that we are leaning towards using more images in our tgui interfaces, not requiring from you but worth noting to anyone else looking at this PR for future inspiration (including me) that a more generic image cache for this stuff is probably warranted.

Reply:
Moved to `datum/plant` with some modifications.